### DISCUSSION TITLE
Make OpenAPI schema for REST API docs available

### DISCUSSION THREAD

**Discussion by nightlark** [community/community#24498] [Apps, API and Webhooks]
<p>I noticed in <a href="https://github.blog/2020-07-02-how-we-launched-docs-github-com/#rest-docs-via-openapi" rel="nofollow noopener">the latest blog post</a> that it says the REST API docs are using an OpenAPI schema.</p>
<p>It would be incredibly useful for projects that implement wrappers for the GitHub REST API for different languages if the latest schema that the docs were generated from was made available, and much better than scraping all of the information as was done with the old REST API doc pages in order to automate some of the maintenance tasks required to keep up to date with all of the API changes that happen.</p>

**Comment by francisfuzz** [comment]
<aside class="quote no-group" data-username="nightlark" data-post="1" data-topic="121057">
<div class="title">
<div class="quote-controls"></div>
 nightlark:</div>
<blockquote>
<p>It would be incredibly useful for projects that implement wrappers for the GitHub REST API for different languages if the latest schema that the docs were generated from was made available, and much better than scraping all of the information as was done with the old REST API doc pages in order to automate some of the maintenance tasks required to keep up to date with all of the API changes that happen.</p>
</blockquote>
</aside>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive. That schema isn’t publicly available as of yet (though we can’t make promises for if or when the team will release it).</p>
<p>However, would you mind <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request? That’s the best place to share requests like these in consideration for future iterations of GitHub features.</p>

**Comment by nightlark** [comment, marked as answer]
<p>It looks like the OpenAPI descriptions are now available in <a href="https://github.com/github/rest-api-description" rel="nofollow noopener">https://github.com/github/rest-api-description</a></p>
<p>The blog post announcing it is <a href="https://github.blog/2020-07-27-introducing-githubs-openapi-description/" rel="nofollow noopener">https://github.blog/2020-07-27-introducing-githubs-openapi-description/</a></p>
